_sass/modules/_footer.scss

```diff
@@ -111,17 +109,18 @@ footer {
   }
 
   .right-links {
-      width: 100%;
-      display: table;
-      margin: 0 0 60px 0;
+    width: 100%;
+    display: table;
```

Comment:
If you have `float: right;`  why you need  `display: table;` here?

Reply:
Done.